numba/typed/typedlist.py

```diff
@@ -26,6 +26,7 @@
 from numba.typed import listobject
 from numba.core.errors import TypingError, LoweringError
 from numba.core.typing.templates import Signature
+from typing import TypeVar, Generic, Iterator
```

Comment:
To stick with style elsewhere should we `import typing as pt` instead?

Reply:
yes! I created this PR before we decided that, and forgot about it. thanks!